bedrock/base/templates/includes/protocol/navigation/menus-refresh/products.html

```diff
@@ -64,7 +64,7 @@ <h2 class="m24-c-menu-item-title" aria-labelledby="fakespot-aria-hint">{{ ftl('n
           </li>
           <li>
             <section class="m24-c-menu-item mzp-has-icon">
-              <a class="m24-c-menu-item-link" href="https://www.thunderbird.net/{{ referrals }}" data-link-text="Thunderbird" data-link-position="topnav - products">
+              <a class="m24-c-menu-item-link" href="https://www.thunderbird.net/{{ utm_params }}" data-link-text="Thunderbird" data-link-position="topnav - products">
```

Comment:
This leads to a 404 URL doe to incorrect formatting, Looks like it needs a `?` before the UTM params
```suggestion
              <a class="m24-c-menu-item-link" href="https://www.thunderbird.net/?{{ utm_params }}" data-link-text="Thunderbird" data-link-position="topnav - products">
```